app/assets/stylesheets/_billables.scss

```diff
@@ -53,6 +53,19 @@
   }
 }
 
+.unbill{
+     background:none!important;
+     border:none;
```

Comment:
Line should be indented 2 spaces, but was indented 5 spaces<br>Colon after property should be followed by one space